src/processes.c

```diff
@@ -512,6 +562,8 @@ static int ps_config(oconfig_item_t *ci) {
   const size_t max_procname_len = MAXCOMLEN - 1;
 #endif
 
+  procstat_t *ps;
```

Comment:
perhaps move this to line 587, so it becomes `procstat_t *ps = ps_list_register(c->values[0].value.string, NULL);`?